core/src/com/mygdx/crazybomber/ui/GameMap.java

```diff
@@ -22,8 +22,8 @@
 import com.mygdx.crazybomber.model.client.CrazyBomberClient;
 
 public class GameMap implements Screen {
-
-    private Sprite background;
+    private GameState game1;
+    private Sprite background,breakableblock,unbreakableblock,bomb; // TODO: extend blocks with sprite
```

Comment:
we should eventually have a sprite for each of the items as well, and item class should also be extended with sprite